pkg/test/framework/components/istio/operator.go

```diff
@@ -934,7 +934,7 @@ func createIstioctlConfigFile(s *resource.Settings, workDir string, cfg Config)
 		cfg.RemoteClusterValues = cfg.ControlPlaneValues
 	}
 
-	configFiles.remoteIopFile = filepath.Join(workDir, "remote.yaml")
+	configFiles.remoteIopFile = filepath.Join(workDir, "external.yaml")
```

Comment:
This change is not correct. `remote.yaml` is not a profile name here, it is the name of the config file that the test framework is using for configuring the test cluster.

Reply:
revert